utils/regex/spec_file_generator.cr

```diff
@@ -0,0 +1,23 @@
+macro generate_source(data)
+<<-EOT
+# THIS FILE IS AUTOMATICALLY GENERATED, DO NOT EDIT !
+
+require "../spec_helper"
+{% for name in data %}
+it "`{{name[0].id}}` ({{name[3].id}}) should match `{{name[2].id}}` ({{name[1].id}})" do
```

Comment:
Sorry, i meant one line instead of 4:

it "`{{name[0].id}}` ({{name[3].id}}) should match `{{name[2].id}}` ({{name[1].id}})" do
    "{{name[1].id}}".match(Emoji::EMOJI_REGEX).try(&.[0]).should eq "{{name[1].id}}"
end